spaceship/spec/client_spec.rb

```diff
@@ -100,4 +112,11 @@ def subject.send_login_request(user, password)
       end
     end
   end
+
+  describe "#log_response" do
+    it 'handles ASCII-8BIT to UTF-8 encoding gracefully' do
+      response = TestResponse.new([130, 5, 3120, 130, 4, 171, 160, 3, 2].pack('C*'))
+      expect(subject.try_log_response(:get, TestClient.hostname, response)).to be_truthy
```

Comment:
Curious why this calls a local `try_log_response` method instead of just `log_response` directly?